netdata.spec.in

```diff
@@ -62,7 +71,7 @@ so that you can get insights of what is happening now and what just
 happened, on your systems and applications.
 
 %prep
-%setup -q -n %{name}-@PACKAGE_VERSION@
+%setup -q
```

Comment:
Yes, have tested with Fedora and CentOS. Read more on the `%setup` macro [here](http://rpm.org/max-rpm-snapshot/s1-rpm-inside-macros.html) to see the defaults.